backend/apps/interactions/services/__init__.py

```
@@ -101,36 +105,69 @@ def subscribe(
+                    existing.save()
+                    return existing
+                else:
+                    # Create new subscription
+                    return Subscription.objects.create(
+                        user=user,
+                        plan_type=plan_type,
+                        expires_at=now + timedelta(days=days),
+                        payment_id=payment_id,
+                        status=SubscriptionStatus.ACTIVE,
+                    )
+        except (django.db.IntegrityError, django.db.DatabaseError, ValueError) as e:
```

Ruff (0.14.13)

Undefined name `django`

(F821)